webapp/src/components/organisms/NavBar/NavBar.tsx

```diff
@@ -7,6 +7,9 @@ export default function NavBar() {
   return (
     <div className="nav-bar">
       <Link to="/" className="nav-bar__logo">
+        <h2>GR RECIPES</h2>
+      </Link>
+      <Link to="/recipe-list" className="nav-bar__link">
         <h2 className="nav-bar__text">GR RECIPES</h2>
       </Link>
       <Link to="/home" className="nav-bar__link">
```

Comment:
I can't comment this far down, but on Line 33 the user name is hardcoded to be Gorden can you set it to the user's actual name?

Also thinking about it more we really use a user ID but let's just pretend the names are unique for vanity URLs in this project 🙈

Reply:
lol true. the only other option I have is to route it with name+userID which will be unique but exposes a backend implementation detail